x/paramfilter/types.go

```diff
@@ -12,4 +12,4 @@ const (
 
 // ErrBlockedParameter is the error wrapped when a proposal to change a
 // blocked parameter is submitted.
-var ErrBlockedParameter = sdkerrors.Register(ModuleName, baseErrorCode, "blocked parameter change, hard fork required")
+var ErrBlockedParameter = sdkerrors.Register(ModuleName, baseErrorCode, "parameter can not be modified")
```

Comment:
the CI will break cause we hardcoded a test, so we should also update

https://github.com/celestiaorg/celestia-app/blob/2dc35d96b11496097c5ae3da226a67b880bb6c97/x/paramfilter/test/param_filter_test.go#L54